src/modules/projects/components/ProjectReferralPostingDetails.tsx

```diff
@@ -0,0 +1,59 @@
+import { Box, Grid, Stack, Typography } from '@mui/material';
+import { ReactNode } from 'react';
+
+import { ReferralPostingStatusDisplay } from './ReferralPostingStatusDisplay';
+
+import { CommonUntyledList } from '@/components/CommonUnstyledList';
+import NotCollectedText from '@/modules/form/components/viewable/item/NotCollectedText';
+import { HmisEnums } from '@/types/gqlEnums';
+import { ReferralPostingDetailFieldsFragment } from '@/types/gqlTypes';
+
+interface Props {
+  referralPosting: ReferralPostingDetailFieldsFragment;
+}
+export const ProjectReferralPostingDetails: React.FC<Props> = ({
+  referralPosting,
+}) => {
+  const bedType = referralPosting?.unitType?.bedType;
+  const col1: Array<[string, ReactNode]> = [
+    [
+      'Referral Status',
+      <ReferralPostingStatusDisplay status={referralPosting.status} />,
+    ],
+    ['Referral Project Name', referralPosting.referredFrom],
+    ['Referral Date', referralPosting.referralDate],
```

Comment:
This should be parsed into the display format using `parseAndFormatDate(referralPosting.referralDate)`